Prac_07/KivyDemos-master/box_layout.kv

```diff
@@ -4,6 +4,14 @@ BoxLayout:
     Button:
         text: 'one'
     Button:
```

Comment:
Try changing this to something the user can use to enter his name and have it displayed (step 8)